script/update-git.js

```diff
@@ -96,9 +106,12 @@ function getPackageDetails(assets, body, arch) {
   }
 }
 
-function updateDependencies(version, packages) {
+function updateDependencies(
+  /** @type {string} */ version,
+  /** @type {Array<{platform: string, arch: string, filename: string, url: string, checksum: string}>} */ packages
+) {
   const dependenciesPath = path.resolve(__dirname, '..', 'dependencies.json')
-  const dependenciesText = fs.readFileSync(dependenciesPath)
+  const dependenciesText = fs.readFileSync(dependenciesPath, 'utf8')
```

Comment:
`tsc` also warned here that `readFileSync` would return a `Buffer` if the encoding wasn't specified. I added this in as a sanity check.